utils/atomic-storybook/project.json

```diff
@@ -0,0 +1,58 @@
+{
+  "targets": {
+    "install:isolated": {
+      "dependsOn": [],
+      "inputs": [],
+      "outputs": [],
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "npm i --package-lock=false",
```

Comment:
Including the lockfiles causes issues with the `commit-generated-files` actions.

Reply:
👎 Keep the lockfile, and modify the action so that the second lockfile is allowed

Reply:
will do